draft-iab-protocol-maintenance.md

```diff
@@ -188,13 +188,24 @@ as quirks in early implementations can affect all subsequent deployments.
 
 # Ecosystem Effects
 
-Once deviations become entrenched, it can be extremely difficult - if not
-impossible - to rectify the situation.
+From observing widely deployed protocols, it appears there are two stable points
+on the spectrum between being strict versus permissive in the presence of
+protocol errors:
 
-Interoperability requirements for protocol implementations are set by other
-deployments.  Specifications and - where they exist - conformance test suites
-might guide the initial development of implementations, but implementations
-ultimately need to interoperate with deployed implementations.
+* If there is a majority of strict implementations, newer implementations will
+  experience failures if they do not conform to the protocol and will fix them
+  before deploying, adding a new strict implementation to the majority.
```

Comment:
```suggestion
  experience failures if they do not comply with protocol requirements. Newer implementations need to
  fix compliance issues in order to be successfully deployed. This ensures
  that most deployments are compliant.
```

I think that compliance > conformance here.  Both arms describe a process that guarantees conformance with group norms.  It's just that one cleaves to a specification that is agreed and written down, the other evolves organically.